### PR TITLE
Metabologram fix

### PR DESCRIPTION
**Pull request by johaGL**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

This PR solves issue #6208

### CODE REVIEW

**Comment by johaGL** [comment]
please check why planemo lint fails @johaGL

**Comment by bgruening** [comment]
> .. WARNING (HelpInvalidRST): Invalid reStructuredText found in help - [<string>:159: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.

**Comment by bgruening** [comment]
If you are using https://github.com/galaxyproject/galaxy-language-server then it might show you the error.

**Comment by johaGL** [comment]
thank you @bgruening with your help I found it :), and solved it. Hope the other tests will pass.

**Comment by johaGL** [comment]
> Please bump the VERSION_SUFFIX number and you might want to update the citation as well.

thank you a lot, yes now it is done :)

**Comment by johaGL** [comment]
thank you a lot for revisions and suggestions, from our side we are done :)

File: tools/dimet/macros.xml [hunk 1]
```diff
@@ -49,14 +49,30 @@
...
+                issn = {1367-4811},
+                doi = {10.1093/bioinformatics/btae282},
+                url = {https://doi.org/10.1093/bioinformatics/btae282},
+                eprint = {https://academic.oup.com/bioinformatics/article-pdf/40/5/btae282/57802928/btae282.pdf},
+            }
+            </citation>
```
Comment (bgruening) [inline tools/dimet/macros.xml:68]:
```suggestion
            <citation type="doi">10.1093/bioinformatics/btae282</citation>
```

**Comment by johaGL** [comment]
thanks! now much better with just doi :)

**Review by bgruening — APPROVED** [review]

**Pull request merged**